doc/nrf/protocols/matter/getting_started/transmission_power.rst

```diff
@@ -0,0 +1,197 @@
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      To build a Matter sample with a custom Bluetooth LE TX power in the nRF Connect for VS Code IDE, add the desired :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig option for the network core to the build configuration's :guilabel:`Extra CMake arguments` and rebuild the build configuration.
+      To build for the network core, make sure to add the ``childImageName_`` parameter between ``-D`` and the name of the Kconfig option.
+      The parameter name varies depending on the devices you are building for.
+      For example:
+
+      * If you want to build for Thread devices for the ``nrf5340dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to 3 dBm, add ``-Dmultiprotocol_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_3=y`` as the CMake argument.
+      * If you want to build for Wi-Fi® devices for the ``nrf7002dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to 3 dBm, add ``-Dhci_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_3=y`` as the CMake argument.
+
+      See `nRF Connect for VS Code extension pack <How to work with build configurations_>`_ documentation for more information.
+
+   .. group-tab:: Command line
+
+      To build a Matter sample with a custom Bluetooth LE TX power from the command line, add the desired :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig option for the network core to the build command.
+      To build for the network core, make sure to add the ``childImageName_`` parameter between ``-D`` and the name of the Kconfig option.
+      The parameter name varies depending on the devices you are building for.
+      For example:
+
+      * If you want to build for Thread devices for the ``nrf5340dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to 3 dBm, run the following command:
+
+        .. code-block:: console
+
+           west build -b nrf5340dk_nrf5340_cpuapp -- -Dmultiprotocol_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_3=y
+
+      * If you want to build for Wi-Fi® devices for the ``nrf7002dk_nrf5340_cpuapp`` build target with a Bluetooth LE TX power equal to 3 dBm, run the following command:
+
+        .. code-block:: console
+
+           west build -b nrf7002dk_nrf5340_cpuapp -- -Dhci_rpmsg_CONFIG_BT_CTLR_TX_PWR_PLUS_3=y
+
+..
+
+.. _ug_matter_gs_transmission_power_wifi:
+
+Changing TX power for Wi-Fi
+***************************
+
+Changing TX power for the Wi-Fi protocol is currently not supported.
+
+The maximum TX power for Wi-Fi depends on the frequency band and the modulation used.
+See `Electrical specification for nRF7002`_ for reference values.
+
+.. _ug_matter_gs_transmission_power_fem:
+
+Changing TX power for FEM
+*************************
+
+Changing TX power for the :ref:`radio Front-End Modules <ug_radio_fem>` is not possible.
+You can only change the FEM gain measured in dB.
```

Comment:
These sentences are a little misleading. 

Maybe just: 

Matter over Thread applications support optional :ref:`radio Front-End Modules <ug_radio_fem>`.
You can control the TX power of the device by configuring the FEM's TX gain.

Reply:
We also include information about nRF21540 under Bluetooth section. Is phrasing that focuses only on 'Matter over Thread' correct?